uaclient/entitlements/tests/test_cc.py

```diff
@@ -61,15 +61,15 @@ class TestCommonCriteriaEntitlementUserFacingStatus:
                 "arm64",
                 "xenial",
                 "16.04 LTS (Xenial Xerus)",
-                "Canonical Common Criteria EAL2 Provisioning is not available"
+                "Common Criteria EAL2 Provisioning Packages is not available"
                 " for platform arm64.\n"
                 "Supported platforms are: x86_64, ppc64le, s390x",
             ),
             (
                 "s390x",
                 "trusty",
                 "14.04 LTS (Trusty Tahr)",
-                "Canonical Common Criteria EAL2 Provisioning is not available"
+                "Common Criteria EAL2 Provisioning Packages is not available"
```

Comment:
This reads strangely to me; it _should_ be "Common Criteria EAL2 Provisioning Packages are not available".